### PR TITLE
hide double checkboxes at edit multiple

### PR DESCRIPTION
**Pull request by zonky2**
you can see this at https://github.com/MetaModels/core/issues/885

### CODE REVIEW

File: contao/templates/dcbe_general_parentView.html5 [hunk 1]
```diff
@@ -9,13 +9,6 @@
 
 				<?php if(strlen($this->breadcrumb)) echo $this->breadcrumb; ?>
 
-				<?php if ($this->select): ?>
```
Comment (discordier) [inline contao/templates/dcbe_general_parentView.html5]: So this removes the "Select all" in parent view at all, why?

Comment (zonky2) [inline contao/templates/dcbe_general_parentView.html5]: it´s double - look at https://github.com/MetaModels/core/issues/885

Comment (discordier) [inline contao/templates/dcbe_general_parentView.html5]: We have:
* dcbe_general_grouping - used in ParentView and ListView when grouping is active.
* dcbe_general_parentView - used in ParentView when no grouping active.
* dcbe_general_listView_sorting - used in ListView when no grouping but sorting is active.
* dcbe_general_listView - used in ListView when neither grouping nor sorting is active.
* dcbe_general_treeview - used in TreeView.

Therefore I simply fail to see where this change fixes MetaModels/core#885 and more important, keeps functionality in parent view.

**Comment by discordier** [comment]
The original issue has been adressed in 1fb11f7

**Pull request closed without merging**